tests/test_alpha.py

```diff
@@ -111,7 +111,7 @@ def test_urls(results):
 
 
 def test_thesis(results):
-    thesis = [{'degree_type': 'PhD Thesis'}]
+    thesis = {'degree_type': 'PhD'}
```

Comment:
I actually stripped `'thesis'` manually since it's not in the schema enum anyway. The list to value part is done via `thesis_out`